### DISCUSSION TITLE
About the github contributors related api

### DISCUSSION THREAD

**Discussion by aayushdhakal** [Tharindu37/ProSolutions#73] [Ideas]
@Tharindu37 currently all the git hub contributors are hard coded so we can use again another single api to call the list of contributors and show their details, so what do you  thik about that ?
API to use:
https://api.github.com/repos/Tharindu37/ProSolutions/contributors

**Comment by Tharindu37** [comment]
@aayushdhakal It's a good idea. I will change the code like this. Thank you.

**Reply by aayushdhakal** [reply]
sure. np

**Reply by therealtmmy** [reply]
How about new contributors? 
Also the site should be hosted on a more secured server.

**Reply by Tharindu37** [reply]
@therealtmmy What are the recommended servers?

**Reply by therealtmmy** [reply]
Use vercel instead.

**Reply by Tharindu37** [reply]
@therealtmmy Oky,

**Comment by therealtmmy** [comment]
How do I add myself to the contributors? It's an api and I dont seems to get that. Why api? I think it would be hard for others to contribute to it that way. What do you think?

**Reply by aayushdhakal** [reply]
you don't just add yourself to the contributors list ,first of all you have to go to issues and solve at least one to be considered contributor to that certain project.
about the api it is the simplest api which is used to pull all the list of contributors from that specific project, you can learn more of that in github docs about api.
why use api .
if there are new contributors added to the project then using api helps to add that users information automatically to the list of the project contributors section

**Reply by therealtmmy** [reply]
Fantastic. Thanks for the easy explaination
